platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/CryptoBenchMerkleDb.java

```diff
@@ -43,4 +43,21 @@ protected VirtualMap<BenchmarkKey, BenchmarkValue> createEmptyMap() {
                 new MerkleDbDataSourceBuilder<>(tableConfig);
         return new VirtualMap<>("vm" + System.nanoTime(), dataSourceBuilder);
     }
+
+    public static void main(final String[] args) throws Exception {
+        final CryptoBenchMerkleDb bench = new CryptoBenchMerkleDb();
+        bench.numFiles = 10;
+        bench.numRecords = 100000;
+        bench.maxKey = 1000000;
+        bench.keySize = 8;
```

Comment:
It makes sense to test with keySize > 8 by default to enable object key mode, as it corresponds to what HS maps actually use.

Reply:
Sure, I'll update the value to 48